custom/icds_reports/utils/aggregation_helpers/distributed/agg_gov_dashboard.py

```diff
@@ -0,0 +1,194 @@
+from dateutil.relativedelta import relativedelta
+
+from custom.icds_reports.const import AGG_GOV_DASHBOARD_TABLE
+from custom.icds_reports.utils.aggregation_helpers import transform_day_to_month, month_formatter
+from custom.icds_reports.utils.aggregation_helpers.distributed.base import BaseICDSAggregationDistributedHelper
+
+
+class AggGovDashboardHelper(BaseICDSAggregationDistributedHelper):
```

Comment:
It would be good if this inherited from the same class as agg child health and followed that model of aggregation/partitioning.

Reply:
do you mean to say to use the temp table aggregate data in that. and then put it back to the real table?
I thought that was just to speed up the aggregation and release the lock. isnt it?

Reply:
@calellowitz  this looks like only major comment out of all. May be its needed/not needed. Could you reply. Like why the partition and using temporary tables is important here.

Reply:
Partitioning is important because it allows us query the parent table for the data we need (`select * from agg_awc where month='2019-12-01' and agg_level=5` rather than `select * from agg_awc_2019-12-01_5`) which means we can use consistent table names in our code when fetching the data no matter the month.
The temp table aspect allows us to keep the old data around (the relevant table from the previous day's agg) which will enable things like rollback if we get to that point.